S/SCIP_PaPILO/build_tarballs.jl

```diff
@@ -0,0 +1,85 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SCIP_PaPILO"
+
+version = v"0.1.0"
+
+sources = [
+    ArchiveSource("https://scipopt.org/download/release/scipoptsuite-v800-rc16.tgz", "0fc025f260e9e77e26e5d218707c04d03492e792c793d34221ada09fbb275b99"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd scipoptsuite*
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix\
+  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}\
+  -DCMAKE_BUILD_TYPE=Release\
+  -DZIMPL=OFF\
+  -DUG=0\
+  -DAMPL=0\
+  -DSYM=bliss\
+  -DIPOPT_DIR=${prefix} -DIPOPT_LIBRARIES=${libdir} ..
+make -j${nproc}
+make install
+
+mkdir -p ../papilo/build
+cd ../papilo/build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+make
+cp bin/papilo ${prefix}/bin/
+
+mkdir -p ${prefix}/share/licenses/SCIP_PaPILO
+for dir in papilo scip soplex; do
+    cp $WORKSPACE/srcdir/scipoptsuite*/${dir}/COPYING ${prefix}/share/licenses/SCIP_PaPILO/LICENSE_${dir}
+done
+cp $WORKSPACE/srcdir/scipoptsuite*/gcg/LICENSE ${prefix}/share/licenses/SCIP_PaPILO/LICENSE_gcg
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(expand_cxxstring_abis(supported_platforms()))
```

Comment:
Did you try building for aarch64-apple-darwin?
```suggestion
platforms = expand_gfortran_versions(expand_cxxstring_abis(supported_platforms(; experimental=true)))
```